engine/src/test/java/io/graphqlcrud/GraphQLSchemaBuilderTest.java

```diff
@@ -64,6 +63,17 @@ public void testSchemaPrint() throws Exception {
             GraphQLFieldDefinition fieldDefinition = objectType.getFieldDefinition("customers");
             Assertions.assertEquals("page",fieldDefinition.getArguments().get(0).getName());
             Assertions.assertEquals("filter", fieldDefinition.getArguments().get(1).getName());
+            if(fieldDefinition.getArgument("filter").getType().toString().contains("CustomerFilterType")) {
+                Assertions.assertTrue(fieldDefinition.getArgument("filter").getType().getChildren().toString().contains("APT_NUMBER"));
+                Assertions.assertTrue(fieldDefinition.getArgument("filter").getType().getChildren().toString().contains("CITY"));
+                Assertions.assertTrue(fieldDefinition.getArgument("filter").getType().getChildren().toString().contains("FIRSTNAME"));
+                Assertions.assertTrue(fieldDefinition.getArgument("filter").getType().getChildren().toString().contains("LASTNAME"));
+                Assertions.assertTrue(fieldDefinition.getArgument("filter").getType().getChildren().toString().contains("ST_ADDRESS"));
+                Assertions.assertTrue(fieldDefinition.getArgument("filter").getType().getChildren().toString().contains("ZIPCODE"));
+                Assertions.assertTrue(fieldDefinition.getArgument("filter").getType().getChildren().toString().contains("PHONE"));
+                Assertions.assertTrue(fieldDefinition.getArgument("filter").getType().getChildren().toString().contains("STATE"));
+                Assertions.assertTrue(fieldDefinition.getArgument("filter").getType().getChildren().toString().contains("SSN"));
+            }
```

Comment:
I meant for BoolenInput, IntInput, StringInput.. here

Reply:
Can't fetch the types here, as they are `TypeReferences` to `StringInput` etc.

Reply:
I mean write test cases for BoolenInput, IntInput, StringInput itself

Reply:
Take a look here - engine/src/test/java/io/graphqlcrud/FilterInputTest.java

Reply:
hmm, sorry I missed that some how looks good.